datafusion/physical-expr/src/aggregate.rs

```diff
@@ -575,18 +576,10 @@ impl AggregateFunctionExpr {
             ReversedUDAF::NotSupported => None,
             ReversedUDAF::Identical => Some(self.clone()),
             ReversedUDAF::Reversed(reverse_udf) => {
-                let mut name = self.name().to_string();
```

Comment:
I think removing this may have other unintended effects. I will request some more eyes on this

Reply:
Thanks!
I agree this code was deliberate & nice. I hope we don't parse those names though.
If there is a better solution to agg reverse causing failures (https://github.com/apache/datafusion/pull/16625#discussion_r2176919070), let me know. I can also drop this fix, I don't like it too.

Alternatively to the fix, I can block reversing for beneficial functions and thus hide the problem for now. Would it be preferred for this PR?